sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/FileDataSourceV2.scala

```diff
@@ -35,4 +40,18 @@ trait FileDataSourceV2 extends TableProvider with DataSourceRegister {
   def fallBackFileFormat: Class[_ <: FileFormat]
 
   lazy val sparkSession = SparkSession.active
+
+  protected def getPaths(map: CaseInsensitiveStringMap): Seq[String] = {
```

Comment:
Although `path` is specific to `FileDataSourceV2`, the logic of this function looks general enough to be at `CaseInsensitiveStringMap` as a utility function. Can we parameterize `"path"` and move this to `CaseInsensitiveStringMap`?

Reply:
IIRC there was a proprosal to offically define array options (not sure it's from @HyukjinKwon or @MaxGekk ). But the problem is, even if we define a general array option getter in the `CaseInsensitiveStringMap`, we still lack a general array option builder.

I'd like to leave this as a special logic in file source, and generalize the array option later.

Reply:
Yep. +1

Reply:
Yup, it was from me and @MaxGekk raised a PR related with that before :D. +1. It's _file_ datasource so it makes sense to me.